modules/apps/object/object-test/src/testFunctional/tests/PicklistExportImport.testcase

```diff
@@ -98,9 +98,60 @@ definition {
 	@description = "LPS-175645 - It’s possible to overwrite Picklist when ERC is duplicated (Update Existing Picklist)"
 	@priority = 5
 	test CanOverwritePicklistWhenERCisDuplicated {
+		task ("Given a Picklist") {
+			Picklist.addPicklistViaAPI(picklistName = "Picklist Test");
 
-		// TODO pending implementation
+			Picklist.gotoPicklists();
+
+			Picklist.gotoPicklistView(picklistName = "Picklist Test");
+
+			SelectFrame(locator1 = "Picklist#IFRAME_PICKLIST_MODAL");
+
+			Type(
+				key_fieldLabel = "External Reference Code",
+				locator1 = "TextInput#GENERIC_TEXT_INPUT",
+				value1 = "testoverwrite001");
+
+			Picklist.clickSaveButton();
+
+			Picklist.gotoPicklists();
+
+			Picklist.gotoPicklistView(picklistName = "Picklist Test");
+
+			Picklist.addPicklistItemViaUI(picklistItemName = "Picklist Item Name Test");
+
+			Picklist.clickSaveButton();
+
+			Picklist.gotoPicklistExport(picklistName = "Picklist Test");
```

Comment:
Does this test need to be exported? Can you help me understand the steps here? My understanding is:

1. Create a picklist
2. Edit its ERC
3. Import a picklist with the same ERC
4. Assert existing picklist gets updated

Not sure if the test even needs to create separate picklist items, but will need that confirmed.

Reply:
Yes, but Im simulating a customer scenario. Since it won't have the dependencies folder, it needs to export the file so it can import it again. But if it's not necessary, I can change it.

Reply:
I see what you're saying. I think for this test though we only need to do the import. The exporting feature should be covered by another automation test so it doesn't need to be included in here too.